lib/msal-browser/src/utils/BrowserConstants.ts

```diff
@@ -13,6 +13,8 @@ export const BrowserConstants = {
     CACHE_LOCATION_LOCAL: "localStorage",
     // Session storage constant string
     CACHE_LOCATION_SESSION: "sessionStorage",
+    // Custom storage constant string
+    CACHE_LOCATION_CUSTOM: "customStorage",
```

Comment:
are we expecting different kinds of storage to come out of this as well to have it be truly custom, or is the intent to just be an in-memory storage?